#!/bin/bash 
#SBATCH --job-name=combine 
#SBATCH --account=commons 
#SBATCH --partition=commons 
#SBATCH --ntasks=1 
#SBATCH --mem-per-cpu=1000m 
#SBATCH --time=00:30:00 
#SBATCH --output=/home/ak308/Bio310/output/ak308_fstscan.out 
#SBATCH --error=/home/ak308/Bio310/output/ak308_fstscan.err

module load GCC/13.3.0
module load VCFtools/0.1.17

VCF=/scratch/ak308/Dog_Selection/Results/KRT71_filtered.recode.vcf
dogpop_curly=/home/ak308/Dog_Selection/Files/KRT71_curlydogs.txt
dogpop_noncurly=/home/ak308/Dog_Selection/Files/KRT71_noncurlydogs.txt
OUT=/scratch/ak308/Dog_Selection/Results

vcftools --gzvcf ${VCF} \
--minQ 30 \
--minDP 3 \
--max-missing 0.8 \
--fst-window-size 2500 \
--fst-window-step 1000 \
--weir-fst-pop ${dogpop_curly} \
--weir-fst-pop ${dogpop_noncurly} \
--out ${OUT}/KRT71_filtered_fst_results_5kbwindow_2.5kbstep

## Make the dog population files 
touch /scratch/ak308/Dog_Selection/Files/KRT71_curlydogs.txt
# copy and paste the names of the dogs in the VCF file that have curly hair 
touch /scratch/ak308/Dog_Selection/Files/KRT71_noncurlydogs.txt
# copy and paste the names of the dogs in the VCf file that do not have curly hair 

# A window based Fst will calculate the average Fst within a 5,000bp segment of the genome. This can help reduce the noise of calculating Fst for every variant, it kind of smooths out the variation, which sometimes can be helpful to see different small genomic regions that are differienitated.  
